go.mod

```diff
@@ -5,29 +5,32 @@ go 1.20
 require (
 	github.com/container-storage-interface/spec v1.6.0
 	github.com/go-logr/logr v1.2.4
+	github.com/go-logr/zapr v1.2.4
 	github.com/golang/protobuf v1.5.3
 	github.com/google/go-cmp v0.5.9
 	github.com/kubernetes-csi/csi-test/v5 v5.0.0
-	github.com/onsi/ginkgo/v2 v2.9.5
-	github.com/onsi/gomega v1.27.7
-	github.com/prometheus/client_golang v1.15.1
+	github.com/kubernetes-csi/external-snapshotter/client/v6 v6.3.0
```

Comment:
external-snapshotter is upgraded to v6.3.3 in `versions.mk`, so it might be better to use v6.3.3.

Reply:
I thought so too. But errors like the below happen when `go mod tidy` with specified v6.3.3.
```
go mod tidy
go: downloading github.com/kubernetes-csi/external-snapshotter/client/v6 v6.3.3
github.com/topolvm/topolvm/e2e tested by
        github.com/topolvm/topolvm/e2e.test imports
        github.com/kubernetes-csi/external-snapshotter/client/v6/apis/volumesnapshot/v1: reading github.com/kubernetes-csi/external-snapshotter/client/go.mod at revision client/v6.3.3: unknown revision client/v6.3.3
github.com/topolvm/topolvm/api/legacy/v1 imports
        k8s.io/apimachinery/pkg/runtime tested by
        k8s.io/apimachinery/pkg/runtime.test imports
        github.com/stretchr/testify/assert: github.com/kubernetes-csi/external-snapshotter/client/v6@v6.3.3: reading github.com/kubernetes-csi/external-snapshotter/client/go.mod at revision client/v6.3.3: unknown revision client/v6.3.3
...
```

I don't know the correct reason but I guessed that `github.com/kubernetes-csi/external-snapshotter/client/v6/` is independent of `github.com/kubernetes-csi/external-snapshotter`. and there are no changes in `client/v6` from v6.3.0. Therefore, there may be `external-snapshotter` v6.3.3 but not `client/v6` v6.3.3.
https://github.com/kubernetes-csi/external-snapshotter/tree/v6.3.3/client

Reply:
🤯

> there are no changes in client/v6 from v6.3.0. 

Anyway I understand it doesn't matter that we select which version. Thank you.